src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/cumulative-metrics-transition-guide.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Transition guide for OpenTelemetry cumulative sum metrics
+metaDescription: 'A transition guide about a change in how New Relic handles OpenTelemetry cumulative sum metrics, and how to update your queries and alerts for that change.'
+redirects: 
+---
+
+Our OTLP metric pipeline is being upgraded to better support how we handle [cumulative metrics](/docs/data-apis/understand-data/metric-data/cumulative-metrics). As part of this upgrade we're transitioning the underlying [metric data type](/docs/data-apis/understand-data/metric-data/metric-data-type/#metric-types) for [monotonic cumulative sums](https://opentelemetry.io/docs/reference/specification/metrics/data-model/#sums) from a `gauge` type to the `cumulativeCount` type. The `cumulativeCount` type allows you to query both the cumulative and delta value associated with reported data. 
+
+Starting March 14, 2023, new New Relic customers will use the new `cumulativeCount` type, but if you were a New Relic customer before that date, you won't have access to that metric type until June 13. On that date, the use of our `gauge` metric type for cumulative sums will be deprecated and reach end-of-life (EoL), and your cumulative sums will be stored as the `cumulativeCount` type. 
+
+If you were a customer using cumulative metrics before March 14, you should check and potentially modify existing queries and alerts that may be relying on those metrics. The sections below will teach you: 
+
+* How to know if your account is impacted
+* How to prepare for the EoL
+* How to preview the `cumulativeCount` functionality 
+
+<Callout variant="important">
+Non-monotonic cumulative sums will **not** be affected by this upgrade.
+</Callout>
+
+## Check your account for impact [#impact]
+
+If your account was sending OTLP cumulative sum metrics to New Relic before March 14, 2023, then that account will be impacted by the EoL on June 13, 2023. 
+
+To check if your account is impacted, run the following NRQL query:
+
+```
+FROM NrIntegrationError SELECT count(*) 
+WHERE newRelicFeature = 'cumulativeSumAsGaugeEoL' 
+SINCE 24 hours ago
+```
+
+The query will return results if you've reported metrics impacted by this EoL in the last 24 hours. 
+
+To investigate further, you can view the event message to see which metrics are affected:
+
+```
+FROM NrIntegrationError SELECT metricName 
+WHERE newRelicFeature = 'cumulativeSumAsGaugeEoL' 
+LIMIT MAX SINCE 24 hours ago
+```
+
+Any monitoring or alerting involving these metrics will need to be reviewed and updated prior to the EoL date. The next section contains suggested actions for impacted metrics. 
+
+## What to do if you're impacted [#what-to-do]
```

Comment:
It sounds like older users won't have access to this new metric type until midnight on June 12th. If you were a user, how would you time this rollout? Do you have to stage these query changes and then roll them out at midnight on June 12th? 

I see later that the doc says older users can access this new metric type by selecting the default delta value. So, are we saying that users could convert all their queries ahead of time and they will work? Are we saying the only difference is how we ingest new sums?

OK, later I see that we're telling users to create side-by-side queries for alerts so that when the switch is flipped, there won't be any impact. Could you make that much clearer earlier?